.circleci/config.yml

```diff
@@ -16,8 +18,119 @@ jobs:
           command: |
             docker version
             docker info
+            docker-compose version
       - run:
           name: "Build image"
           command: docker build --progress=plain -t notary_client .
-      - run: ./buildscripts/circle_parallelism.sh
-      - run: docker-compose -f docker-compose.yml down -v && docker-compose -f docker-compose.rethink.yml down -v
+      - run:
+          name: "ci"
+          command: docker run --rm -e NOTARY_BUILDTAGS --env-file buildscripts/env.list --user notary notary_client bash -c "make ci && codecov"
+      - run:
+          name: "Teardown"
+          command: docker-compose -f docker-compose.yml down -v && docker-compose -f docker-compose.rethink.yml down -v
+
+  job_02:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: ~/go/src/github.com/theupdateframework/notary
+    environment:
+      NOTARY_BUILDTAGS: none
+      DOCKER_BUILDKIT: 1
+    steps:
+      - add_ssh_keys
+      - checkout
+      - run:
+          name: "Docker Info"
+          command: |
+            docker version
+            docker info
+            docker-compose version
+      - run:
+          name: "Build image"
+          command: docker build --progress=plain -t notary_client .
+      - run:
+          name: "ci"
+          command: docker run --rm -e NOTARY_BUILDTAGS --env-file buildscripts/env.list --user notary notary_client bash -c "make ci && codecov"
```

Comment:
`NOTARY_BUILDTAGS=none`?

EDIT: ah ,ok, you define it over there.

Reply:
Ah, yes, I initially did that as I expected it to show up in the CI UI (but I think I mixed up CircleCI with TravisCI - Travis shows those in the UI 😂)

Still thought it was nice to have all the env-vars defined in a single place